sonar-flex-plugin/src/test/java/org/sonar/plugins/flex/FlexSquidSensorTest.java

```diff
@@ -28,6 +28,9 @@
 import org.junit.Rule;
 import org.junit.Test;
 import org.mockito.Mockito;
+import org.mockito.internal.creation.MockSettingsImpl;
```

Comment:
should not be part of this commit

Reply:
Moved to the commit where versions are updated